test/functionalTest/cases/3112_ngsiv1_autocast/create_attributes_with_autocast.test

```diff
@@ -0,0 +1,353 @@
+# Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Create attributes using NGSIv1 autocast functionality
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0 IPV4 -ngsiv1Autocast
+
+--SHELL--
+
+#
+# 01. Create entity with Quantity, Number, Boolean, DateTime and ISO8601 types
+# 02. Get entity using NGSIv2 and check values are rigth
```

Comment:
right :)

Reply:
Fixed in f65e5f29a1e53753c25d1c2e81ea7e3277fb241c